include/uv-os390.h

```diff
@@ -26,5 +26,9 @@
 
 #define UV_PLATFORM_LOOP_FIELDS                                               \
   void* ep;                                                                   \
+  int fs_event_msg_queue;                                                     \
+
+#define UV_PLATFORM_FS_EVENT_FIELDS                                           \
+  char rfis_rftok[8];                                                         \
```

Comment:
I think the OP means that it changes the ABI by making the structure larger, so is ABI incompatible